artifacts/definitions/Windows/Detection/Yara/memory.yaml

```diff
@@ -1,26 +1,34 @@
-name: Windows.Detection.ProcessMemory.CobaltStrike
+name: Windows.Detection.Yara.Process
+author: Matt Green - @mgreen27
 description: |
-  This artifact scans processes for a Cobalt Strike yara signature and enables 
-  fast upload of processes with hits for analysis
+  This artifact enables running Yara over processes in memory.
+    
+  There are 3 kinds of Yara rules that can be deployed:   
+   &nbsp;1. Url link to a yara rule.   
+   &nbsp;2. Shorthand yara in the format "wide nocase ascii:string1,string2,string3".  
+   &nbsp;3. or a Standard Yara rule attached as a parameter.  
+  Only one method of Yara will be applied and search order is as above. The 
+  default is Cobalt Strike opcodes.
   
-  Commonly abused by attackers for its high feature / low cost return on 
-  investment, Cobalt Strike is a commercial penetration testing tool providing 
-  post exploitation capability throughout the attack lifecycle.
-
-author: "Matt Green - @mgreen27"
-
-reference:
-  - https://attack.mitre.org/software/S0154/
-  - https://malpedia.caad.fkie.fraunhofer.de/details/win.cobalt_strike
-
+  Regex parameters can be applied for process name and pid for targeting. The 
+  artifact also has an option to upload any process with Yara hits.  
+  
+  Note: the Yara scan will stop after one hit.
+  
+type: CLIENT
 parameters:
   - name: ProcessRegex
```

Comment:
we seem to have lost the whitelist from above? this could be useful

Reply:
I removed it as I never really used it and post processed whitelists. Do you think its worth adding back in?

Reply:
I think it is useful - sometimes we need to avoid scanning sensitive processes (like lssas.exe) because it will also trigger EDRs as well.

Reply:
just added whitelist capability in with placeholder path.